pkg/apis/core/v1/helper/helpers.go

```diff
@@ -370,3 +370,62 @@ func ScopedResourceSelectorRequirementsAsSelector(ssr v1.ScopedResourceSelectorR
 	selector = selector.Add(*r)
 	return selector, nil
 }
+
+// NodeSelectorRequirementsAsLabelRequirements converts the NodeSelectorRequirement
+// type to a labels.Requirement type.
+func NodeSelectorRequirementsAsLabelRequirements(nsr v1.NodeSelectorRequirement) (*labels.Requirement, error) {
```

Comment:
does this func need to be exported?

Reply:
No this one doesn't, modified the name to `nodeSelectorRequirementsAsLabelRequirements`